modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/index.js

```diff
@@ -228,7 +228,12 @@ export async function loadData(
 	page = 1,
 	sorting = []
 ) {
-	const url = new URL(apiURL, themeDisplay.getPortalURL());
+	const fullUrl = apiURL.startsWith('/')
```

Comment:
Should we support this case? Is not an option to enforce the use of full/working apiUrls directly from the backend?

Reply:
Yeah, I agree with you, this is a bit smelly. Ideally, we should not support relative URL. But, some things to have in mind:
- Every single `getAPIURL()` in DXP currently returns a relative URL. We have 17 of them.
- We are supporting relative URLs in [fetch](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/fetch.es.js#L31-L46). This PR technically improves consistency. We should probably remove that support as well, but that change may go beyond FDS.

I think it's best we first tackle the issue of the ticket, URLs incorrectly built in backend. We can consider dropping support for relative URLs in a separate task.

Reply:
Thanks for the explanation

> We can consider dropping support for relative URLs in a separate task.

Yes, this should be the way to do this